src/scss/components/_toast.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Toast Component
+ * =================================
+ * Used to display text in a pop-up style notification.
+ *
+ * *The `c-toast` component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding `@include toast();` into your SCSS dependencies file.*
+ *
+ * Documentation:
+ * https://fozzie.just-eat.com/styleguide/ui-components/toast
+ */
+
+$toast-radius           : 8px !default;
```

Comment:
Is the `!default` used anywhere atm?

Reply:
Was left in after rebasing v1 🤦‍♂️  Will update.